packages/flutter/test/widget/positioned_test.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
```

Comment:
We usually put a blank line between the `dart:` imports and the `package:` imports.  I believe that's in some official Dart style guide somewhere.